inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,86 +1,412 @@
+Differences are highlighted. You can accept an annotation by clicking on
+it. 
+** As a curator, you can also create new annotations on this page. It works exactly like on the Annotation page. Note that users who have nothing but curation rights do not see nor have access to the annotation page (see xref:User_rights[User Rights]).
+
+[[knowledge_bases_in_getting_started_in_structrue]]
+* *Knowledge Base*: Also see xref:knowledge_bases_in_getting_started[the note-box on knowledge bases]. On the Knowledge Base page, you can manage and create your
+knowledge base(s) for the project you are in. You can create new knowledge bases from the scratch, modify them and integrate existing knowledge bases into your project which are either local (that is, they are saved on your device) or remote (that is, they are online). Note that this knowledge base page is distinct from the tab of the same name in the project settings (see xref:knowledge_bases_in_getting_started[Knowledge Base in section Project Settings]).
+
+NOTE: *Agreement:* The annotations of different annotators usually do not match perfectly. This aspect of difference / similarity is called agreement. For agreement, the measures provided in {product-name}, precision, recall and F1 are common.
+
+* *Agreement*: On this page, you can calculate the so called annotator agreement.
+*_Note:_* Only documents *marked as finished* by annotators (clicking on the
+little lock on the annotation page) are taken into account.
+
+* *Monitoring:* Here you can check the overall progress of your
+project; see which user is working on or has finished which document; and toggle for each user the status of each document between *Done / In Progress* or between *New / Locked*. For details, see <<Monitoring>> in the main documentation.
+
+* *Evaluation*: The evaluation page shows a learning courve diagram of
+each recommender (see xref:recommenders_in_getting_started[Recommender]).
+
+* *Settings*: Here, you can organize, manage and adjust all the details of your project. We had a look on those you need to get started for your own projects in the section <<Project Settings>> already.
+
+This was the overview on what you can _do_ in each project and what elements each project _has_. Now you are ready to go for your own annotations.
+
+
+== First Annotations with {product-name}
+
+
+In this section, we will make first annotations. If you have not downloaded and imported an example project yet, we recommend to return to <<Download and Import an Example Project>> and do so first.
+In this section, no or little theory and background will be explained. In case you want to have some theory and background knowledge first, we recommend reading the section <<Structure Of an Annotation Project>>.
+
+
+*Create your first annotations*
+
+You may want to watch our https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s[tutorial video „Overview“] on how to create
```

Comment:
I would swap these sentence parts, first say this guide guides you then for additional info watch this video